src/shallow.ts

```diff
@@ -4,16 +4,4 @@ import { shallow } from './vanilla/shallow.ts'
 // export { shallow } from './vanilla/shallow.ts'
 // export { useShallow } from './react/shallow.ts'
```

Comment:
Please use these lines.

Reply:
I'll make this change. Originally, I hadn't used these lines to keep the PR small, but I will make this change.

Reply:
ah, sorry about the confusion.